ui/lib/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum PageRoute {
```

Comment:
My preference is for "RESTy" stuff :p but again no big deal, we can adopt this pattern if you prefer it